testcases/kernel/controllers/cpuctl/cpuctl_def_task02.c

```diff
@@ -77,7 +77,7 @@ extern void cleanup(void)
 
 volatile int timer_expired = 0;
 
-int main(int argc, char *argv[])
+int main(int argc LTP_ATTRIBUTE_UNUSED, char *argv[] LTP_ATTRIBUTE_UNUSED)
 {
```

Comment:
The C standard allows main to be defined as `int main(void)' as well, which would be a cleaner solution here.